kernel/classes/datatypes/ezobjectrelationlist/ezobjectrelationlisttype.php

```diff
@@ -1564,15 +1579,21 @@ function metaData( $contentObjectAttribute )
                 $attributes = $content['temp'][$subObjectID]['attributes'];
             else
             {
-                $subObjectVersion = $relationItem['contentobject_version'];
-                $object = eZContentObject::fetch( $subObjectID );
+                $subObjectVersionNum = $relationItem['contentobject_version'];
+                $subObject = eZContentObject::fetch( $subObjectID );
+
+                // Using last version of object (version inside xml data is the original version)
+                $subCurrentVersionObject = $subObject->currentVersion();
+                if( $subCurrentVersionObject instanceof eZContentObjectVersion ) {
```

Comment:
nitpick: space after the if  bracket in new line (even dunno if we are more relaxed with this in legacy...)